apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.html

```diff
@@ -16,23 +16,28 @@ <h1 class="page-subtitle">{{'VO_DETAIL.MEMBERS.TITLE' | translate}}</h1>
         {{'VO_DETAIL.MEMBERS.ADD_MEMBER' | translate}}
       </button>
     </div>
-    <button
-      *ngIf="inviteAuth"
-      [disabled]="blockManualMemberAdding"
-      [matMenuTriggerFor]="menu"
-      class="me-2 dropdown-toggle"
-      color="accent"
-      mat-flat-button>
-      {{'VO_DETAIL.MEMBERS.INVITE' | translate}}
-    </button>
-    <mat-menu #menu="matMenu">
-      <button *ngIf="inviteAuth" (click)="onInviteMember()" class="action-button" mat-menu-item>
-        {{'VO_DETAIL.MEMBERS.INVITE_ONE' | translate}}
-      </button>
-      <button *ngIf="inviteAuth" (click)="onBulkInvite()" class="action-button" mat-menu-item>
-        {{'VO_DETAIL.MEMBERS.INVITE_BULK' | translate}}
+    <span
+      [matTooltipDisabled]="!inviteDisabled"
+      [matTooltipPosition]="'above'"
+      matTooltip="{{'GROUP_DETAIL.MEMBERS.INVITE_DISABLED' | translate}}">
```

Comment:
nitpick - rest of the translated text is duplicated for VO_DETAIL, so would be consistent to do so here as well.